### ISSUE TITLE
It didn't work , How to add the config.json ?

### ISSUE THREAD

**Issue by Tony-li1991xx** [mylesgray/homebridge-blueair#44]
![微信截图_20210515155204](https://user-images.githubusercontent.com/40227889/118353229-ed3fc280-b597-11eb-8d81-7753b5befbaa.png)

  I put the config-example.json into the config . but it didn't work 
Please tell me what's wrong with it
![微信截图_20210515154706](https://user-images.githubusercontent.com/40227889/118353310-44459780-b598-11eb-9de6-cbbdc337787f.png)
![微信截图_20210515154736](https://user-images.githubusercontent.com/40227889/118353315-4c053c00-b598-11eb-8947-9386918bd3b2.png)

**Comment by mylesagray** [comment]
Your username and password are invalid.

**Comment by Tony-li1991xx** [comment]
I had try ,I write the right user&password, but still can't work .
What else could be the problem？
![微信截图_20210518145002](https://user-images.githubusercontent.com/40227889/118605610-0be8c800-b7e9-11eb-9c78-e6fe7a907d13.png)
![微信截图_20210518145129](https://user-images.githubusercontent.com/40227889/118605647-13a86c80-b7e9-11eb-8211-b0f17f8a5082.png)

**Comment by Tony-li1991xx** [comment]
not work


![微信截图_20210518150017](https://user-images.githubusercontent.com/40227889/118606463-06d84880-b7ea-11eb-86eb-4babe15f82df.png)

**Comment by mylesagray** [comment]
Is your airPurifierIndex actually 2? Do you have more than one on your account?

**Comment by Tony-li1991xx** [comment]
no, just one accound, could you give me a post/get API to test my accound whether can get  data from the blueair sever.

**Comment by mylesagray** [comment]
Details are at the bottom of the readme: https://github.com/mylesagray/homebridge-blueair#api-research

**Comment by downtempo** [comment]
@mylesagray, can you please provide the Postman Environment variables? In the README under "Postman collection and environment", the image which is supposed to show these variables gives a "Not Found" error when I try to load it: https://camo.githubusercontent.com/7501ca6830d936cc05fac172c89c43c1cda358b03acbcd94c3dcfdbf09c330ff/68747470733a2f2f646c2e64726f70626f7875736572636f6e74656e742e636f6d2f732f7975336c7a377234377065303065782f53637265656e73686f74253230323031382d30332d303425323031352e30362e31302e706e673f646c3d30

**Comment by mylesagray** [comment]
@downtempo - Did realise it was busted! I've fixed that now and here it is for reference:

![Env Vars](https://dl.dropbox.com/s/ogzizxgz1rpyhen/Screenshot%202021-05-19%20at%2012.04.25.png)

**Comment by kjohn** [comment]
> no, just one accound, could you give me a post/get API to test my accound whether can get data from the blueair sever.

Did you end up getting it working? What did you do?

**Comment by fjs21** [comment]
Please try https://github.com/fjs21/homebridge-blueair

The plug-in is configured using Homebridge UI.
